Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -54,17 +54,9 @@ extension File {
 }
 
 public enum AccessControlLevel: String {
-    case Private = "private"
```

Comment:
I didn't realize that, but indeed it looks like this is the case in the init method of `MissingDocsRule` where the configuration is mapped over. I'll change the rawValue back and add a new init `fromSourcekitValue`. It's worth noting however: `AccessControlLevel` seems to be inconsistent with how it's done in other parts of the code. `KindType` and `SwiftDeclarationKind` both use the sourcekit value strings as their rawValue.